calico-enterprise/operations/nftables.mdx

```diff
@@ -0,0 +1,206 @@
+apiVersion: kubeproxy.config.k8s.io/v1alpha1
+kind: KubeProxyConfiguration
+mode: nftables
+```
+:::
+
+## Before you begin
+
+**Required**
+
+- A Linux host that meets the following requirements:
+
+  - x86-64, arm64, ppc64le, or s390x processor
+  - 2CPU
+  - 2GB RAM
+  - 10GB free disk space
+  - Linux kernel version 5.13 or later with `nft` >= 1.0.1
+
+- $[prodname] can manage `cali` and `tunl` interfaces on the host
+
+  If NetworkManager is present on the host, see [Configure NetworkManager](../operations/troubleshoot/troubleshooting.mdx#configure-networkmanager).
+
+## How to
+
+### Create a single-host Kubernetes cluster with the nftables kube-proxy enabled.
+
+1. [Follow the Kubernetes instructions to install kubeadm](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/)
+
+   :::note
+
+   After installing kubeadm, do not power down or restart
+   the host. Instead, continue directly to the next step.
+
+   :::
+
+1. As a regular user with sudo privileges, open a terminal on the host that you installed kubeadm on.
+
+1. Create a kubeadm configuration file that enables the nftables kube-proxy mode.
+
+   ```bash
+   cat > config.yaml <<EOF
+   apiVersion: kubeadm.k8s.io/v1beta4
+   kind: InitConfiguration
+   ---
+   kind: ClusterConfiguration
+   apiVersion: kubeadm.k8s.io/v1beta4
+   kubernetesVersion: v1.31.0
+   proxy: {}
+   networking:
+     podSubnet: "192.168.0.0/16"
+   ---
+   apiVersion: kubeproxy.config.k8s.io/v1alpha1
+   kind: KubeProxyConfiguration
+   mode: nftables
+   EOF
+   ```
+
+   :::note
+
+   If 192.168.0.0/16 is already in use within your network you must select a different pod network
+   CIDR, replacing 192.168.0.0/16 in the above configuration.
+
+   :::
+
+1. Initialize the control plane using the following command.
+
+   ```bash
+   sudo kubeadm init --config=config.yaml
+   ```
+
+1. Execute the following commands to configure kubectl (also returned by `kubeadm init`).
+
+   ```bash
+   mkdir -p $HOME/.kube
+   sudo cp -i /etc/kubernetes/admin.conf $HOME/.kube/config
+   sudo chown $(id -u):$(id -g) $HOME/.kube/config
+   ```
+
+### Install $[prodname] in nftables data plane mode
+
+1. Install the Tigera Operator and custom resource definitions.
+
+   ```bash
+   kubectl create -f $[filesUrl]/manifests/operator-crds.yaml
+   kubectl create -f $[filesUrl]/manifests/tigera-operator.yaml
+   ```
+
+   :::note
+
+   Due to the large size of the CRD bundle, `kubectl apply` might exceed request limits. Instead, use `kubectl create` or `kubectl replace`.
+
+   :::
+
+1. Install $[prodname] by creating the necessary custom resource. For more information on configuration options available in this manifest, see [the installation reference](../reference/installation/api.mdx).
+
+   ```bash
+   cat > custom-resources.yaml <<EOF
+   apiVersion: operator.tigera.io/v1
+   kind: Installation
+   metadata:
+     name: default
+   spec:
+     calicoNetwork:
```

Comment:
This is going to need some tweaks for enterprise - namely, it needs to set the `variant` to TigeraSecureEnterprise, and also include image pull secrets. You can nab the content from `custom-resources.yaml` and just add `linuxDataplane: Nftables` to it. 

We may just want to instruct the user to download `custom-resources.yaml` and make that change themselves, as there are a number of additional objects needed in that file for enterprise.